qutebrowser/config/websettings.py

```diff
@@ -238,6 +238,25 @@ def _set(self, value, qws=None):
         self._setter(*args)
 
 
+class ThirdPartyCookies(Base):
+
+    """The ThirdPartyCookiePolicy setting is different from other settings."""
+
+    mapping = (
```

Comment:
This should probably be called `MAPPING` (and `REVERSE_MAPPING`, see below) as it's a constant, and these are all-caps by convention